### ISSUE TITLE
not seem work on .wav

### ISSUE THREAD

**Issue by djmisterjon** [Dijji/FileMeta#56]
Hi guy, i just try on **mp3** and it work fine, so first thanks a lot for this amazing tool !!

I just get a issue, and want know it am do something wrong ?
If i try on .wav file, it seem not work. ?
![image](https://user-images.githubusercontent.com/24865815/56434022-f834a580-62a0-11e9-95d3-d70c01dc59ed.png)
Any tips, am doing something wrong ?

**Comment by Dijji** [comment]
No, I'm having the same problem. Recent versions of Windows have been showing an annoying habit of bypassing the registry settings for some media files, e.g. MKV, and loading the supplied Microsoft property handler directly from Explorer. This makes it impossible for File Meta to intervene. In looks as though .WAV needs to be added to the list.

Dijji

**Comment by Dijji** [comment]
I have now investigated what is going on here. The results are described [here](../wiki/Windows-10-and-media-files). My results are consistent with yours, and include a description of what you can expect for now, and what File Meta is doing next.
